make/common/NativeCompilation.gmk

```diff
@@ -129,8 +129,10 @@ include native/ToolchainDefinitions.gmk
 #
 SetupNativeCompilation = $(NamedParamsMacroTemplate)
 define SetupNativeCompilationBody
-  # In this functions, helpers named Setup<Foo> are just setting variables.
-  # Functions called Create<Foo> are setting up rules to create files.
+  # In this functions, macros named Setup<Foo> are just setting variables.
```

Comment:
"functions" is still grammatically wrong. Do you actually mean "this macro", "these macros" or "this file"? I think there were a couple of other mentions of "function" further down as well.

Reply:
Better now?